lib/models/Ride.dart

```diff
@@ -548,3 +569,198 @@ class RecurringRidesGenerator {
     );
   }
 }
+
+Widget onTheWayRide(context, Ride ride) {
+  String startLocation = ride.startLocation;
+  DateTime startTime = ride.startTime;
+  String timeString = DateFormat.jm().format(startTime);
+  return Row(children: <Widget>[
+    Expanded(
+      child: RichText(
+        text: new TextSpan(
+          style: new TextStyle(
+            fontSize: 20.0,
+            color: Colors.black,
+          ),
+          children: <TextSpan>[
+            new TextSpan(text: 'Head to the '),
+            new TextSpan(
+                text: 'pickup location\n',
+                style: new TextStyle(fontWeight: FontWeight.bold)),
+            new TextSpan(text: 'for @$startLocation by $timeString'),
+          ],
+        ),
+      ),
+    )
+  ]);
+}
+
+Widget arrivedRide(context, Ride ride) {
+  String startLocation = ride.startLocation;
+  return Row(children: <Widget>[
+    RichText(
```

Comment:
Expanded around here for overflow and it should be good

Reply:
fixed